hack/vendor-flux-manifests.sh

```diff
@@ -30,4 +31,14 @@ do
     vendor "$var"
 done
 
+info "downloading distro repository"
+curl -sLO https://github.com/controlplaneio-fluxcd/distribution/archive/refs/heads/main.tar.gz
+tar xzf main.tar.gz -C "${DEST_DIR}"
+
+mkdir -p "${IMG_DIR}"
```

Comment:
maybe should add a check here?

Reply:
`-p` will not create the dir if it exists.